src/pages/use-cases/bandwidth-optimization.jsx

```diff
@@ -0,0 +1,78 @@
+  imageSrc: BandWidthImage1,
+  imageAlt: 'bandwidth illustration',
+};
+
+const sectionContent1 = {
+  title: 'Optimal Bandwith Management with Cilium’s Bandwidth Manager',
+  description:
+    'Cilium’s Bandwidth Manager allows rate-limiting per Pod with just one line of YAML. Compared to other alternatives, the Bandwidth Manager provides a 4x reduction in latency, ensuring a smooth network experience and isn’t detrimental to performances and is designed for multi-queue and multi-core NICs. ',
+  imageSrc: BandWidthImage2,
+  imageAlt: 'Cilium bandwidth manager illustration',
+};
+
+const sectionContent2 = {
+  title: 'Accelerate network performance with Cilium BBR',
+  description:
+    'Cilium supports BBR, a congestion control algorithm developed by Google, making it the first platform to do so. Google observed up to 2,700x improvement in throughput when testing BBR, making it a valuable addition for optimizing network performance. Cilium’s BBR provides exceptional improvements in throughput and latency for external-facing applications, offering consumers a delightful user experience',
+  imageSrc: BandWidthImage3,
+  imageAlt: 'Cilium bandwidth manager illustration',
+  whiteBackground: true,
+  imageRight: false,
+};
+
+const sectionContent3 = {
+  title: '100Gbit/S Clusters With Cilium: Building Tomorrows Networking',
+  description:
+    "Cilium BIG TCP allows for larger packets than the traditional 64KB limit by leveraging IPv6's Hop-by-Hop header, which can specify payload lengths up to 512KB. This is particularly useful for organizations building networks capable of 100Gbps and beyond. BIG TCP does not require modifying the MTU on network devices, making it easier to implement than Jumbo Frames. With BIG TCP, Cilium offers enhanced network performance for nodes, enabling users to extract as much performance as possible from the network.",
+  videoSrc: 'https://www.youtube.com/embed/Kvdh78TURck',
+};
+
+const testimonials = [
+  {
+    logo: <IsovalentLogo />,
+    title: 'BIG Performances with BIG TCP on Cilium',
+    CTAtext: 'Read The Blog Post',
+    url: 'https://isovalent.com/blog/post/big-tcp-on-cilium/',
+    description:
+      'Many of the organizations adopting Cilium – cloud providers, financial institutions and telecommunications providers – all have something in common: they all want to extract as much performance from the network as possible and they are constantly looking out for marginal performance gains.',
+  },
+];
+
+const BandwidthLatencyPage = () => (
+  <MainLayout>
+    <section className="bg-[#F6F7F8]">
+      <IntroSection {...introContent} />
+      <ImageFeatureSection {...sectionContent1} />
+      <ImageFeatureSection {...sectionContent2} />
+      <VideoFeatureSection {...sectionContent3} />
+      <UseCaseCard
+        heading="Learn More About CIlium’s New Big TCP feature"
```

Comment:
```suggestion
        heading="Learn More About CIlium’s Big TCP Feature"
```

Reply:
The text is transformed to capitalize the first letter of each word. so the specific casing doesn't matter here.